plugins/type_converter/help.md

```diff
@@ -11,6 +11,10 @@ for enabling input interoperability between certain plugins.
 
 _This plugin does not contain any requirements._
 
+# Supported Product Versions
+
+_There are no supported product versions listed._
+
```

Comment:
Just a general question, but wondering if this is required?

Reply:
It got added when I regenerated. Is it good practice to remove this?

Reply:
I'll have a look through and replicate for each of the other actions too. Good spot.